### PR TITLE
Gmail bugfix send mail

### PR DESCRIPTION
**Pull request by avidan-H**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/content/pull/8167

## Description
Fixes a bug where in Gmail integration where send-mail command would send both body and htmlBody

## Screenshots
Before:
image
After:
image

## Minimum version of Demisto
- [x] 5.0.0
- [x] 5.5.0
- [x] 6.0.0

## Does it break backward compatibility?
   - [x] Yes
       - It's possible that there are some customers that may expect the incorrect behavior when sending emails with both `body` and `htmlBody` arguments...

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by avidan-H** [comment]
@richardbluestone can you please review the release notes?
 - [ ] [Packs/Gmail/ReleaseNotes/1_0_4.md](https://github.com/demisto/content/edit/gmail-bugfix-send-mail/Packs/Gmail/ReleaseNotes/1_0_4.md?pr=%2Fdemisto%2Fcontent%2Fpull%2F8519)

File: Packs/Gmail/Integrations/Gmail/Gmail.py [hunk 1]
```diff
@@ -1670,7 +1670,7 @@ def send_mail(emailto, emailfrom, subject, body, entry_ids, cc, bcc, htmlBody, r
         # if there is only body and no attachments to the mail , we would like to send it without attaching every part
         message = MIMEText(body, 'plain', 'utf-8')  # type: ignore
     else:
-        message = MIMEMultipart()  # type: ignore
+        message = MIMEMultipart('alternative') if body and htmlBody else MIMEMultipart()  # type: ignore
```
Comment (Itay4) [inline Packs/Gmail/Integrations/Gmail/Gmail.py:1673]: what happens if we just set it to `MIMEMultipart('alternative')` if not both provided ?

Comment (avidan-H) [reply Packs/Gmail/Integrations/Gmail/Gmail.py:1673]: I'll test it now.

Comment (avidan-H) [reply Packs/Gmail/Integrations/Gmail/Gmail.py:1673]: Still seems to work. I tested it with,
1. `body` and `htmlBody`,
2. just `body`,
3. just `htmlBody`.
Regarding all the other command arguments I haven't checked.

Comment (Itay4) [reply Packs/Gmail/Integrations/Gmail/Gmail.py:1673]: ok lets keep it as is

**Pull request merged**